src/v2/Utils/Hooks/useTimer.tsx

```diff
@@ -28,12 +28,14 @@ export const useTimer = (endDate: string, startAt: string = ""): Timer => {
   const duration = Duration.fromISO(
     DateTime.fromISO(endDate).diff(DateTime.fromISO(currentTime)).toString()
   )
+
   const hasEnded = Math.floor(duration.seconds) <= 0
 
   const timeBeforeStart = Duration.fromISO(
-    DateTime.fromISO(startAt).diff(DateTime.fromISO(currentTime)).toString()
+    DateTime.fromISO(currentTime).diff(DateTime.fromISO(startAt)).toString()
```

Comment:
this is the part that I want to verify needs to be changed and fixed, possibly in a separate PR

Reply:
Yea, pull out this kind of update in a separate PR!

Reply:
agree on a separate PR for this.